[easy] Print disk availability and meminfo at start of job

ttung opened the pull request:
### Description
This might be helpful for debugging.

Removed a bit of cruft from past debugging runs.

### Test plan
YOLO

jgadling commented on .happy/terraform/modules/sfn_config/nextstrain.wdl:
```diff
@@ -43,6 +43,9 @@ task nextstrain_workflow {
     set -Eeuxo pipefail
     shopt -s inherit_errexit
 
+    df
```
gotta add `1>&2` so miniwdl doesn't eat stdout!

ttung replied:
damn it you're right.

ttung replied:
ok fixed.

Commit:
Print disk availability and meminfo at start of job

### Description
This might be helpful for debugging.

Removed a bit of cruft from past debugging runs.

### Test plan
YOLO
